content/en/security_platform/application_security/troubleshooting.md

```diff
@@ -0,0 +1,444 @@
+---
+title: Troubleshooting Application Security Monitoring
+kind: documentation
+further_reading:
+- link: "/security_platform/getting_started/"
+  tag: "Documentation"
+  text: "Get Started Instrumenting Your Application to Send Signals"
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Monitoring Threats with Datadog Application Security"
+- link: "/security_platform/application_security/setup_and_configure/#compatibility"
+  tag: "Documentation"
+  text: "Programming Language and Framework Compatibility"
+---
+
+If you experience unexpected behavior with Datadog Application Security, there are common issues you can investigate, as mentioned below. If you continue to have trouble, reach out to [Datadog support][1] for further assistance. 
+
+## Application Security data is not appearing in Datadog
```

Comment:
The difference between this section and the "Application Security events are missing in the dashboard" is fuzzy to me. It seems to address the same issue at different level

Reply:
I consolidated the two sections so it's all under "No Suspicious Requests detected by Application Security".